### PR TITLE
[WIP] Trying to improve coverage

### PR DESCRIPTION
**Pull request by csabadenes**

### CODE REVIEW

File: tests/test_admin.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 patch_admin_extra_urls_decorators()
 
 from unicef_security import admin
-from unicef_security.models import User, Region
+from unicef_security.models import User, Region, BusinessArea
```
Comment (domdinicola) [inline tests/test_admin.py]: please use isort

File: tests/test_admin.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+from tests.fixtures import patch_admin_extra_urls_decorators
+patch_admin_extra_urls_decorators()
+
+from unicef_security import admin
+from unicef_security.models import User, Region, BusinessArea
+
+
+def test_admin_reverse():
+    model = User
+    page="changelist"
```
Comment (domdinicola) [inline tests/test_admin.py]: flake

File: tests/test_azure.py [hunk 3]
```diff
@@ -1,29 +0,0 @@
-import os
-from pathlib import Path
-
-import pytest
```
Comment (domdinicola) [inline tests/test_azure.py]: what happened here?

File: tests/test_admin.py (hunk 2 above)
Comment (csabadenes) [reply tests/test_admin.py]: Ok, i will fix the flake errors

File: tests/test_admin.py (hunk 1 above)
Comment (csabadenes) [reply tests/test_admin.py]: Ok, i will fix these.
Is there a reason why isort is not called on the tests directory in the tox.ini file? (same question for flake8)

File: tox.ini [hunk 4]
```diff
@@ -1,5 +1,6 @@
 [tox]
-envlist = py{36}-d{21,22}
+# envlist = py{36}-d{21,22}
+envlist = py{36}-d{21}
```
Comment (domdinicola) [inline tox.ini]: why this?

File: tests/test_tasks.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+# @mock.patch('celery.app.default_app.task')
+# @override_settings(CELERY_TASK_ALWAYS_EAGER=True)
+# @override_settings(CELERY_ALWAYS_EAGER=True)
+# def test_sync_business_area(celery_worker, monkeypatch):
+# @pytest.mark.skip(reason="maybe figure out how to `delay` the task..")
+@pytest.mark.usefixtures('depends_on_current_app')
+def test_sync_business_area(monkeypatch):
+    '''
+    with celery_app.conf.update(CELERY_ALWAYS_EAGER=True):
```
Comment (domdinicola) [inline tests/test_tasks.py]: you can override this in tests

File: tests/test_tasks.py [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+        monkeypatch.setitem('celery.app.default_app', 'task', lambda: a)
+        monkeypatch.setattr('celery.app.default_app', 'task', lambda: a, raising=False)
```
Comment (domdinicola) [inline tests/test_tasks.py]: using too much monkeypatches

**Pull request merged**